install/kubernetes/cilium/values.yaml

```diff
@@ -510,6 +510,36 @@ daemon:
   # -- Configure where Cilium runtime state should be stored.
   runPath: "/var/run/cilium"
 
+  # -- Configure a custom list of possible configuration override sources
+  # The default is "config-map:cilium-config,cilium-node-config,node"
+  # Possible entries are
+  #   - "configmap:<namespace>/<name>" -- a ConfigMap with name <name>, optionally at a namespace.
+  #   - "cilium-node-config:<namespace>/<name>" --  any or all CiliumNodeConfigurations. If <name> is specified,
+  #                                                 then only that one is considered. Otherwise, all CNCs in the given namespace
+  #                                                 that match the LabelSelector are used
+  #   - "node" -- Any annotations on the Node object of the form "io.cilium.config/<KEY> = <VALUE>"
+  #
+  # Note that this value should be a comma-separated string.
+  configSources: ~
+
+  # -- allowedConfigOverrides is a list of config-map keys that can be overridden.
+  # That is to say, if this value is set, config sources (excepting the first one) can
+  # only override keys in this list.
+  #
+  # This takes precedence over blockedConfigOverrides.
+  #
+  # By default, all keys may be overridden. To disable overrides, set this to an invalid key.
```

Comment:
Is there an example invalid key that you could suggest? It's not so immediately obvious to me how I would disable this. For that matter, I'd probably just suggest to change `configSources` if the user wanted to completely disable overrides.

Reply:
Someone suggested '*', but this is even easier.